templates/includes/card-group-record-summary-no-image.html

```diff
@@ -8,19 +8,19 @@
 
 <li class="col-sm-12 col-md-6 col-lg-4">
     <div class="card-group-record-summary">
-        <a href="{% record_url record is_editorial=True %}" class="card-group-record-summary__link" data-card-type="card-group-record-summary-no-image" data-card-title="{{ record.title }}"{% if FEATURE_RECORD_LINKS_GO_TO_DISCOVERY %} target="_blank"{% endif %}>
+        <a href="{% record_url record is_editorial=True %}" class="card-group-record-summary__link" data-card-type="card-group-record-summary-no-image" data-card-title="{% firstof description_override|safe record.description|striptags|linebreaks %}"{% if FEATURE_RECORD_LINKS_GO_TO_DISCOVERY %} target="_blank"{% endif %}>
             <p class="card-group-record-summary__heading">
                 {% if record.reference_number %}
                 <span class="sr-only">
                     {{ record.reference_number }} -
                 </span>
                 {% endif %}
-                {{ record.title }}
+                {% firstof description_override|safe record.description|striptags|linebreaks %}
```

Comment:
Sorry @gtvj, I hadn't quite realised that we're already in a `<p>` tag here, so we don't need `|linebreaks`.

Just want to check that the intention is definitely to use `record.description` here instead of `record.title`? descriptions can be pretty darn long in some cases... it seems strange to be using the full value as a 'sort of heading', but then always displaying a truncated version below.

Reply:
All sorted and reverted back to `record.title`. I made this change because I wasn't getting anything back for `record.title` and in most cases it's simply a truncated `record.description` anyway. Still, it's using `record.title` now (with `record.description` appearing below).